crates/services/gas_price_service/src/v0/tests.rs

```diff
@@ -91,27 +116,28 @@ impl MetadataStorage for ErroringMetadata {
     }
 }
 
+fn arb_config() -> V0AlgorithmConfig {
```

Comment:
Nit: While we already use the `arb` abbreviation for the `arb_metadata()`, I find it easier to read if we spell out the full word:
```suggestion
fn arbitrary_config() -> V0AlgorithmConfig {
```